library.rb

```diff
@@ -85,6 +80,13 @@ def load_from_file(class_name)
       end
     end
 
+    def popular_in_order_by(field, counter = nil)
+      grouped_by = @orders.group_by { |e| e.send(field) }
+      sort_fields = grouped_by.keys.sort { |x,y| grouped_by[y].count <=> grouped_by[x].count }
+      return sort_fields.first(counter) if counter
+      sort_fields.select { |e| grouped_by[e].count == grouped_by[sort_fields.first].count }
```

Comment:
1. According to task it should 1, but ok, let left it as it is
2. Yes, I know. That's why it is enough to use.sort_by { |name, orders| orders.size } or even max_by